Classes/OGVPlayerState.m

```diff
@@ -154,12 +156,14 @@ -(BOOL)paused
 
 -(float)playbackPosition
 {
-    // @todo use alternate clock provider for video-only files
+    double position = 0.0;
     if (playing) {
-        return audioFeeder.playbackPosition + initialAudioTimestamp;
+        position = CACurrentMediaTime() - initTime + offsetTime;
```

Comment:
Using a clock unrelated to the audio clock means audio is out of sync. Need to always use the audio clock if there is audio.